### ISSUE TITLE
Google maps not working with new nightly build environment

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#5348]
With 2015.12.31-NB Google Maps is not working when opening the live map (no tiles are loading).

Logcat shows:
```
14:51:13.449 Warning System.err 21208  IOException processing: 26
14:51:13.449 Warning System.err 21208  java.io.IOException: Server returned: 3
14:51:13.449 Warning System.err 21208  	at android_maps_conflict_avoidance.com.google.googlenav.map.BaseTileRequest.readResponseData(BaseTileRequest.java:115)
14:51:13.449 Warning System.err 21208  	at android_maps_conflict_avoidance.com.google.googlenav.map.MapService$MapTileRequest.readResponseData(MapService.java:1473)
14:51:13.449 Warning System.err 21208  	at android_maps_conflict_avoidance.com.google.googlenav.datarequest.DataRequestDispatcher.processDataRequest(DataRequestDispatcher.java:1117)
14:51:13.449 Warning System.err 21208  	at android_maps_conflict_avoidance.com.google.googlenav.datarequest.DataRequestDispatcher.serviceRequests(DataRequestDispatcher.java:994)
14:51:13.449 Warning System.err 21208  	at android_maps_conflict_avoidance.com.google.googlenav.datarequest.DataRequestDispatcher$DispatcherServer.run(DataRequestDispatcher.java:1702)
14:51:13.449 Warning System.err 21208  	at java.lang.Thread.run(Thread.java:818)
```

**Comment by Eselter** [comment]
I can confirm.
--- System information ---
Device: LG-D802 (g2_open_com, lge)
Android version: 5.0.2
Android build: LRX22G
Cgeo version: 2015.12.31-NB-655619e
Google Play services: enabled
Low power mode: active
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: false
Map strategy: detailed
HW acceleration: enabled (default state)
System language: pl_PL
Geocaching sites enabled:
 - geocaching.com: logged in (Nazwa użytkownika OK) / Basic
Installed cgeo plugins:  none
--- End of system information ---

**Comment by MrYaz** [comment]
Still no Google maps are loaded in the NBs

**Comment by Lineflyer** [comment]
That is not very surprising as this issue is still open.

**Comment by MrYaz** [comment]
Hehe... You are right...
What I actually was meaning with my comment was:
What is the current progress with this issue?
Has someone examined the problem?
With the NB:s generating each night again the only reason not to try them out for me personally is the current lack of Google maps.

**Comment by rsudev** [comment]
We have examined the problem (using the wrong api key in the build) and hopefully in the next days I (or someone else) will have the time to extend the new grade build to cater for this special situation on the build machine.

**Comment by MrYaz** [comment]
Great news @rsudev !
Thanks for the update and your work!

**Comment by Bananeweizen** [comment]
@rsudev Now that I read your comment I remember some strange thing from the old build script. There are actually 2 files with keys on the server, and one of them was used only for releases, the other for everything else (or similar). I totally forgot this when creating the gradle build. And I don't really know why these 2 files exist to begin with. I assume that we can always use the release key. If you agree to that, we just need @mucek4 to modify the file in the protected area on the server. As a workaround we could patch the keys.xml during build.

**Comment by mucek4** [comment]
I have edited and set both keys to same value.

**Comment by samueltardieu** [comment]
I've launched a built for -NB1.

**Comment by Lineflyer** [comment]
Confirmed fixed in 2016.01.11-NB1
